db/compaction_picker.cc

```diff
@@ -1275,6 +1294,41 @@ bool LevelCompactionPicker::PickCompactionBySize(VersionStorageInfo* vstorage,
   return inputs->size() > 0;
 }
 
+bool LevelCompactionPicker::PickIntraL0Compaction(
+    VersionStorageInfo* vstorage, CompactionInputFiles* inputs) {
+  inputs->clear();
+  const std::vector<FileMetaData*>& level_files =
+      vstorage->LevelFiles(0 /* level */);
+  if (level_files.size() == 0 || level_files[0]->being_compacted) {
```

Comment:
I'm thinking about how we can avoid to trigger the test too easily.
For example, we just have 4 files while there happens to be a L1->L2 compaction going on, we don't want compact these 4 files immediately.

Can we raise the standard slightly. Just random thought. How about only triggering it if number of L0 files is no fewer than trigger+2, or trigger * 1.5?